src/styles/__variables.scss

```
@@ -0,0 +1,5 @@
+$transition-time: 300ms;
+$nav-item-color: #060b35;
+$nav-link-hover: #00acdc;
+$card-hover: #34568b;

```

Variable names should be obvious.
```suggestion
$card-hover-color: #34568b;

```